[bitnami/postgresql-repmgr]  After a 3-node cluster, simultaneous restart of all nodes resulted in failure to start properly.

### Name and Version

bitnami/postgresql-repmgr:15.5.0-debian-11-r15

### What architecture are you using?

amd64

### What steps will reproduce the bug?


1. deploy a 3 node postgres cluster  in k8s v1.23, deploy in static pod mode
2. check it work right, there are one primary and two standby
3. stop postgres in each node, almost at the same time
4. check each postgres has stop
5. start each postgres, almost at the same time
6. check each postgres status 

### What is the expected behavior?

The postgres cluster can recover into healthz

### What do you see instead?

each pg pod in crash, can not election the primary

pg pod exit log
```
[2024-05-31 18:54:06] [ERROR] unable connect to upstream node (ID: 2337678), terminating
[2024-05-31 18:54:06] [HINT] upstream node must be running before repmgrd can start
[2024-05-31 18:54:06] [INFO] repmgrd terminating...
```

### Additional information

```
wal_log_hints = 'on'

reconnect_attempts='24'
reconnect_interval='5'

node_rejoin_timeout=300
```

--- a/bitnami/postgresql-repmgr/15/debian-12/rootfs/opt/bitnami/scripts/librepmgr.sh
+++ b/bitnami/postgresql-repmgr/15/debian-12/rootfs/opt/bitnami/scripts/librepmgr.sh
@@ -237,8 +237,12 @@ repmgr_get_primary_node() {
                 primary_port="$REPMGR_PRIMARY_PORT"
             fi
         else
-            primary_host="$upstream_host"
-            primary_port="$upstream_port"
+           if  [[ "${upstream_host}:${upstream_port}" = "${REPMGR_NODE_NETWORK_NAME}:${REPMGR_PORT_NUMBER}" ]];  then
+                info "Skip Primary Ip self. Starting PostgreSQL normally..."
+           else
+                primary_host="$upstream_host"
+                primary_port="$upstream_port"
+           fi
         fi
     fi
 

--- a/bitnami/postgresql-repmgr/15/debian-12/rootfs/opt/bitnami/scripts/librepmgr.sh
+++ b/bitnami/postgresql-repmgr/15/debian-12/rootfs/opt/bitnami/scripts/librepmgr.sh
@@ -237,12 +237,12 @@ repmgr_get_primary_node() {
                 primary_port="$REPMGR_PRIMARY_PORT"
             fi
         else
-           if  [[ "${upstream_host}:${upstream_port}" = "${REPMGR_NODE_NETWORK_NAME}:${REPMGR_PORT_NUMBER}" ]];  then
-                info "Skip Primary Ip self. Starting PostgreSQL normally..."
-           else
+            if  [[ "${upstream_host}:${upstream_port}" = "${REPMGR_NODE_NETWORK_NAME}:${REPMGR_PORT_NUMBER}" ]];  then
+                info "Avoid setting itself as primary. Starting PostgreSQL normally..."
+            else
                 primary_host="$upstream_host"
                 primary_port="$upstream_port"
-           fi
+            fi
         fi
     fi
 

